templates/cookiecutter-github-action-two-stages-pipeline/{{cookiecutter.cicd_folder_name}}/pipeline.yaml

```diff
@@ -0,0 +1,139 @@
+name: Pipeline
+
+on: [push]
+
+env:
+  aws_access_key_id_: ${{ secrets.{{cookiecutter.deployer_aws_access_key_id_variable_name}} }}
+  aws_secret_access_key_: ${{ secrets.{{cookiecutter.deployer_aws_secret_access_key_variable_name}} }}
+  sam_template: {{cookiecutter.sam_template}}
+  testing_stack_name: test-stack
+  prod_stack_name: prod-stack
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          # uncomment and modify the following step for running the unit-tests
+          # Assuming python runtime
+          # pip install pytest
+          # pip install -r /path/to/requirements.txt
+          # python -m pytest /path/to/unit-tests
+  
+  build-and-deploy-feature:
+    if: startsWith(github.ref, 'refs/heads/feature')
+    needs: [test]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Load init-env-vars.sh
+        run: . {{cookiecutter.cicd_folder_name}}/init-env-vars.sh
```

Comment:
there is no before_script in gh action, and each job has its own environment, so they need to load separately